tutorials/tutorial02_multiapps/doc/content/step03_coupling.md

```diff
@@ -0,0 +1,114 @@
+# Step03 Coupling
+
+!---
+
+## MultiApp Coupling
+
+Now we need to combine these capabilities to perform coupled calculations.
+
+Knowing that it is trivially easy for MOOSE-based calculations to couple to Auxiliary Variables or Postprocessor values - it should be obvious how this is going to work:
+
+1. Develop standalone calculations that have stand-in values in Auxiliary fields that are coupled to.
+2. Put them together into a MultiApp hierarchy
+3. Utilize Transfers to move values from one App to another to fill the Auxiliary fields with "real" values
+4. ...
+5. Profit!
+
+All that is left to explore is how we *control* such a calculation.
+
+!---
+
+## Loose Coupling
+
+!row!
+!col! width=50%
+Remembering back to the intro - the simplest coupling is "Loose".  Here we will simply exchange values once per timestep and move on.
+
+This is easily achieved with MultiApps and Transfers.  By default, the MultiApp will be executed at the correct moment during a timestep (as dictated by `execute_on`) and the Transfers associated with that MultiApp are also executed at that time.  Then the simulation will move on to the next timestep.
+
+No problem!
+
+!col-end!
+
+!col width=50%
+!media images/coupling.png
+       style=width:80%;margin-left:auto;margin-right:auto;display:block;
+
+!row-end!
+
+!---
+
+## Loose Coupling Example (01_master.i)
+
+Here we continue with the microstructure calculation.  But now we will add Kernels and MaterialProperties that couple to the transferred fields.
+
+The Master app can be thought of as computing a "source" term - that is then used as a forcing function in the sub-apps.
+
+The sub-apps then compute a "material property"... which is then "upscaled" to the Master app via a PostprocessorInterpolationTransfer.
+
+What we end up with is then a smooth field over the whole Master domain that represents "Diffusvity" - which is then fed as the "Diffusivity" into the diffusion equation being solved by the Master app.
+
+!---
+
+## Run 01_master.i
+
+!listing step03_coupling/01_master.i
+         caption=01_master.i
+
+!---
+
+## Picard
+
+Loose coupling, though easy and stable, has problems with accuracy.  Since data is only exchanged once per timestep there is the very real possibility that the solutions in both apps are not at an equilibrium with eachother.  This is especially true as timesteps grow larger.
```

Comment:
```suggestion
Loose coupling, though easy and stable, may not be accurate.  Since data is only exchanged once per timestep there is the very real possibility that the solutions in both apps are not at an equilibrium with each other.  This is especially true as timesteps grow larger.
```